package.json

```diff
@@ -17,6 +17,7 @@
   "scripts": {
     "start": "gulp",
     "build": "npm run clean && gulp release && zip -r rivet.zip css js sass tokens index.html",
```

Comment:
Do we need to zip this stuff up in the `build` script anymore since we there's a separate step/script that being used to zip the file that gets attached to the release?

The only thing we use that .zip file for is to attach to the GitHub release, so I"m not sure the zip file creation needs to be apart of this `build` step. But, I could be wrong.